user/installing-dependencies.md

```diff
@@ -254,20 +254,82 @@ addons:
 
 ## Installing Packages on OS X
 
-To install packages that are not included in the [default OS X environment](/user/reference/osx/#Compilers-and-Build-toolchain) use [Homebrew](http://brew.sh) in your `.travis.yml`. For example, to install beanstalk:
+To install packages that are not included in the [default OS X environment](/user/reference/osx/#Compilers-and-Build-toolchain) use [Homebrew](http://brew.sh) and our Homebrew addon in your `.travis.yml`. For example, to install beanstalk:
 
 ```yaml
-before_install:
-  - brew update # see note below
-  - brew install beanstalk
+addons:
+  homebrew:
+    packages:
+    - beanstalk
+```
+{: data-file=".travis.yml"}
+
+By default, the Homebrew addon will not run `brew update` before installing packages. `brew update` can take a long time and slow down your builds. If you need more up-to-date versions of packages than the snapshot on the build VM has, you can add `update: true` to the addon configuration:
+
+```yaml
+addons:
+  homebrew:
+    packages:
+    - beanstalk
+    update: true
+```
+{: data-file=".travis.yml"}
+
+### Taps and Casks
```

Comment:
Perhaps split this section into "Casks" and "Taps". The header is "Taps and Casks", but Casks is treated first. Seems strange.